server/core/uno.py

```
@@ -0,0 +1,215 @@
+        return self.value == 'draw-four'
+
+    def is_wild(self) -> bool:
+        return self.value == 'wild'
+
+    def __repr__(self) -> str:
+        return f'Card(color={self.color}, value={self.value})'
+
+
+class Deck:
+    SHUFFLE_FREQ = 50
+    COLORS = ['red', 'blue', 'green', 'yellow']
```

Ruff (0.13.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)